main/ui/src/main/resources/i18n/strings.properties

```diff
@@ -108,6 +108,15 @@ unlock.error.heading=Unable to unlock vault
 unlock.error.invalidMountPoint.notExisting=Mount point "%s" is not a directory, not empty or does not exist.
 unlock.error.invalidMountPoint.existing=Mount point "%s" already exists or parent folder is missing.
 
+# Lock
+## Force
+lock.forced.heading=Forcefully lock vault?
+lock.forced.message=The vault "%s" cannot be locked due to open files or pending operations. You can enforce locking, but unsaved data will be lost and pending read/write operations aborted.
+lock.forced.confirmBtn=Force locking
```

Comment:
```suggestion
lock.forced.heading=Graceful lock failed
lock.forced.message=Locking "%s" was blocked due to open files or pending operations. You may force lock this vault, however interrupting I/O may result in the loss of unsaved data.
lock.forced.confirmBtn=Force Lock
```

Reply:
The title should inform instead of asking questions. After all, the software should serve the user not the other way round.

Reply:
But on the other hand, the user need to take an action. Also, the user wanted to lock the vault, but it failed. Hence, the title should ask the user. I would suggest to replace "may" by "can", because the user is not "allowed" to do it, but just a possible action.

Well, we certainly cannot have both. I'll let you decide

Other question: Why capitalize the words for the button? From the moment three word are used it looks kinda strange...